anomalib/models/draem/utils/augmenter.py

```diff
@@ -21,7 +21,7 @@
 from torch import Tensor
 from torchvision.datasets.folder import IMG_EXTENSIONS
 
-from anomalib.models.draem.utils.perlin import rand_perlin_2d_np
+from anomalib.utils.generators import rand_perlin_2d_np
```

Comment:
we also have `anomalib.data.utils` directory? Would it be an idea to move this there ?

Reply:
Yeah. That's a good idea.